proxy/proxypair.js

```diff
@@ -179,17 +176,13 @@ class ProxyPair {
     if (this.webrtcIsReady()) {
       this.client.close();
     }
-    this.client = null;
```

Comment:
Why are we deleting this (and the following three) initialisation? I'm not saying we shouldn't, I'm just trying to understand the code.